src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -159,24 +161,33 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
         rewardsamountCryptoBaseUnit: opportunityData?.rewardsAmountsCryptoBaseUnit?.[0] ?? '0',
       })
 
+      if (bn(withdrawBps).isZero()) return
+
       const quote = await getThorchainSaversWithdrawQuote({ asset, accountId, bps: withdrawBps })
 
       const { dust_amount, expected_amount_out } = quote
-      const withdrawFee = amountCryptoThorBaseUnit
-        .minus(expected_amount_out)
-        .div(bn(10).pow(THOR_PRECISION))
-
-      const dustAmountCryptoPrecision = fromThorBaseUnit(dust_amount)
 
-      setWithdrawFee(withdrawFee.toFixed())
-      setDustAmount(dustAmountCryptoPrecision.toFixed(asset.precision))
+      setWithdrawFeeCryptoBaseUnit(
```

Comment:
`withdrawFee` and `dustAmount` were missing the `baseUnit` terminology, and giving it the same name as the state field would cause shadowing on l. 145